asciidoc/courses/app-go/modules/1-driver/lessons/6-type-system/questions/1-node-property.adoc

```diff
@@ -0,0 +1,11 @@
+[.question]
+= 1. Accessing Node Properties
+
+Which of the following options is a valid method for accessing the `name` property on the node object `node`?
```

Comment:
note: in the latest 5.x, you have the generic/type-safe `neo4j.GetProperty[T PropertyValue](entity Entity, key string) (T, error)`

Reply:
applies to both nodes and relationships